spec/models/certificate_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe Certificate, type: :model do
+  it "is valid with valid attributes" do
+    expect(Certificate.new(usage: 'signing', value: 'Test')).to be_valid
```

Comment:
would also be good to test when the usage is `blah` i.e. check we're not allowing non-allowed usage types